includes/Core/Util/Activation.php

```diff
@@ -155,9 +155,81 @@ private function get_activation_notice() {
 					ob_start();
 					?>
 					<script type="text/javascript">
-					if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
-						sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
-					}
+						if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
+							sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
+						}
+
+						jQuery(document).ready(function() {
+							var trackingScriptPresent = !! googlesitekit.admin.trackingOptIn;
+
+							if ( googlesitekit.admin.trackingOptIn ) {
+								jQuery('#opt-in').attr( 'checked', googlesitekit.admin.trackingOptIn );
+							}
+							if ( googlesitekit.admin.proxySetupURL ) {
+								jQuery( '#start-setup-link' ).attr( 'href', googlesitekit.admin.proxySetupURL );
+							}
+
+							// TODO: Fix this; it doesn't currently work as
+							// `sendAnalyticsTrackingEvent` is not synchronous
+							// but fails to implement an async API that confirms
+							// the event was sent. This was moved from an
+							// `onClick` handler, but did not work there either.
+							jQuery( '#start-setup-link' ).on( 'click' , function() {
+								if ( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
+									sendAnalyticsTrackingEvent( 'plugin_setup', 'goto_sitekit' );
+								}
+							} );
+
+							jQuery('#opt-in').on( 'change' , function( event ) {
+								if ( event.target.disabled ) {
+									event.preventDefault();
+									return;
+								}
+
+								var checked = event.target.checked;
+
+								var body = {
+									googlesitekit_tracking_optin: checked,
+								};
+								var self = this;
+
+								jQuery(self).attr( 'disabled', true );
```

Comment:
An element's disabled state should be toggled with `.prop` instead of `,attr`

Reply:
Sorry, I forgot how to jQuery!